Sources/Playdate/Documentation.docc/DownloadingTheTools.md

```diff
@@ -18,6 +18,24 @@ To build games for the Playdate, you will need to download a Swift toolchain and
 4. You should see the following window if the Swift nightly toolchain was installed successfully: 
 @Image(source: "swift-installation-successful", alt: "A screenshot of the Swift nightly toolchain package installer success screen.")
 
+5. The installed Swift Toolchain can be found under `/Library/Developer/Toolchains` when **Install for all users on this computer** was selected, or under `$HOME/Library/Developer/Toolchains` when **Install for me only** was selected.
+
+```console
+$ find /Library/Developer/Toolchains ~/Library/Developer/Toolchains -depth 1 -print 2>/dev/null
```

Comment:
Can we add a small prelude to this snippet with something like: "You can find all installed toolchains in both directories with the following command"

Reply:
Good point, I'll add it in 👍